tsa-test/src/test/resources/args/receive_bounce_msg.fc

```diff
@@ -0,0 +1,7 @@
+
+
```

Comment:
Please don't add blank lines in the beginning of the file (or add stdlib import).